src/llamafactory/model/loader.py

```diff
@@ -103,12 +106,13 @@ def load_tokenizer(model_args: "ModelArguments") -> "TokenizerModule":
         setattr(processor, "video_resolution", model_args.video_resolution)
         setattr(processor, "video_fps", model_args.video_fps)
         setattr(processor, "video_maxlen", model_args.video_maxlen)
-    except Exception:
+    except Exception as e:
+        logger.warning("Failed to load processor. Error: {}".format(e))
         processor = None
 
     # Avoid load tokenizer, see:
     # https://github.com/huggingface/transformers/blob/v4.40.0/src/transformers/models/auto/processing_auto.py#L324
-    if "Processor" not in processor.__class__.__name__:
+    if processor and "Processor" not in processor.__class__.__name__:
```

Comment:
in Line 111, processor is set to None...

Reply:
`None.__class__.__name__ == "NoneType"`

Reply:
Well, it is reasonable to add a if-else here